crates/holochain/src/core/queue_consumer/tests.rs

```diff
@@ -259,9 +259,15 @@ async fn publish_loop() {
         timer.elapsed() >= Duration::from_secs(60) && timer.elapsed() < Duration::from_secs(61)
     );
 
-    publish_dht_ops_workflow(db.clone(), dna_network.clone(), ts.clone(), author.clone())
-        .await
-        .unwrap();
+    publish_dht_ops_workflow(
+        db.clone(),
+        dna_network.clone(),
+        ts.clone(),
+        author.clone(),
+        MIN_PUBLISH_INTERVAL,
```

Comment:
Given the above about tuning params / defaults, may be acceptable to define a test constant for this?

Reply:
I've used the `default()` now in all places. Do you prefer a specific test constant set to the same as the default instead?